string ROS_System
string action
string object
float32 x
float32 y
float32 z
string arm
---
string answer